docs/submission/submitting-molecular-data.md

```diff
@@ -75,211 +73,19 @@ To do this, change directories into `conf` folder and open the `application.prop
-    "read_length_r2": 150,
-    "insert_size": 298,
-    "sample_barcode": null,
-    "library_name": "Pond-147580"
-  }
-],
-```
-
-#### **Files:**
-
-The files section contains metadata about the molecular files to be submitted. Sequencing data of both `BAM` and `FASTQ` type files is accepted.
-
-- There is no special requirement for FASTQ files except that paired end data should have the reads in two FASTQ files (one for each end).
-- ARGO does not accept interleaved FASTQ files.
-- Compression of FASTQ files is **required**; both _gzip_ (suffix .fq.gz or .fastq.gz) or _bz2_ (suffix .fq.bz2 or .fastq.bz2) are supported.
-- For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
-
-The fields include:
-
-| Payload Field | Attribute                                               | Description                                                                                 | Permissible Values |
-| ------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------- | ------------------ |
-| fileName      | ![Required](/assets/submission/dictionary-required.svg) | Name of the file.                                                                           |                    |
-| fileSize      | ![Required](/assets/submission/dictionary-required.svg) | Size of the file, in bytes.                                                                 |                    |
-| fileMd5sum    | ![Required](/assets/submission/dictionary-required.svg) | Compute the md5sum of the file. This must match what is computed when the file is uploaded. |
-| fileType      | ![Required](/assets/submission/dictionary-required.svg) | Data format of sequencing files.                                                            | BAM, FASTQ         |
-| fileAccess    | ![Required](/assets/submission/dictionary-required.svg) | The permission level of a file.                                                             | open, controlled   |
-| dataType      | ![Required](/assets/submission/dictionary-required.svg) | Descriptor of the type of file being submitted.                                             | Submitted Reads    |
-
-**Example files portion of payload:**
-
-```json
-"files": [
-  {
-    "fileName": "test_rg3.bam",
-    "fileSize": 14911,
-    "fileMd5sum": "178f97f7b1ca8bfc28fd5586bdd56799",
-    "fileType": "BAM",
-    "fileAccess": "controlled",
-    "dataType": "Submitted Reads"
-  }
-]
-```
-
-### Step 1. Prepare Song sequencing_experiment metadata
-
-This is an example of a correctly formatted `sequencing_experiment` metadata submission in JSON format, according to the rules presented above:
+### Step 1. Prepare molecular metadata sequencing_experiment payload
+Before proceeding, please read the instructions on how to [prepare and validate](/docs/submission/molecular-data-prep) molecular metadata payloads.

```

Comment:
be sure to update this link with the correct page (right now it's linking to the first version)

Reply:
this is already one on the new netlify